### PR TITLE
Added pagination count if use single field.

### PR DESCRIPTION
**Pull request by hrach-o**

### CODE REVIEW

File: src/Pagination/Footer/Footer.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import Buttons from './Buttons/Buttons'
 import Numbers from './Buttons/Numbers'
 
-const Footer = ({ paginationButtons, ...props }) => {
+const Footer = ({ paginationButtons, isSingleField, ...props }) => {
   const { type } = paginationButtons
 
-  if (type === 0)
-    return <Buttons paginationButtons={paginationButtons} {...props} />
+  if (type === 0 || isSingleField)
```
Comment (Tiko-Nazaryan) [inline src/Pagination/Footer/Footer.js:8]: Why we need this condition here ?

File: src/index.js [hunk 2]
```diff
@@ -53,6 +54,10 @@ const Builder = ({
           pagination={isPagination ? pages[currentPage].fields : []}
           logic={isLogicOn ? logic : []}
         />
+        {isSingleField && <div className='boomForm-paginationCount'>
+          {currentPage + 1}/{pages.length || 1}
+        </div>
+        }
```
Comment (Tiko-Nazaryan) [inline src/index.js:60]: This should be shown when the pagination is turned on too

File: src/index.js [hunk 3]
```diff
@@ -15,7 +15,8 @@ const Builder = ({
   fields = [],
   button = {},
   pagination = {},
-  logic
+  logic,
+  isSingleField = false
```
Comment (Tiko-Nazaryan) [inline src/index.js:19]: Lets get this parameter from pagination object 'Cause it's related to pagination

File: src/index.js (hunk 2 above)
Comment (Tiko-Nazaryan) [reply src/index.js:60]: Let's change isSingleField to smth like pageCounter. 'Cause builder don't have a clue what is single Field

File: src/index.js [hunk 4]
```diff
@@ -33,7 +34,7 @@ const Builder = ({
   const [currentPage, setCurrentPage] = useState(initial)
 
   useEffect(() => {
-    setCurrentPage(initial)
+    setCurrentPage(initial || 0)
```
Comment (Tiko-Nazaryan) [inline src/index.js:37]: Explain please why we need this condition here ?

File: src/Pagination/Footer/Footer.js [hunk 5]
```diff
@@ -2,11 +2,11 @@ import React from 'react'
...
+    return <Buttons paginationButtons={paginationButtons} isSingleField={isSingleField} {...props} />
```
Comment (Tiko-Nazaryan) [inline src/Pagination/Footer/Footer.js:9]: We don't need isSingleField here

File: src/Pagination/Footer/Buttons/Buttons.js [hunk 6]
```diff
@@ -46,7 +47,7 @@ const Buttons = ({
 
   return (
     <>
-      {captcha !== undefined && currentPage === pages.length - 1 && (
+      {!isSingleField && captcha !== undefined && currentPage === pages.length - 1 && (
```
Comment (Tiko-Nazaryan) [inline src/Pagination/Footer/Buttons/Buttons.js:50]: isSingleField is useless here cause this current condition is fully covers all stuff related condition

File: src/Pagination/Footer/Footer.js (hunk 1 above)
Comment (hrach-o) [reply src/Pagination/Footer/Footer.js:8]: If using single field Add-ons, builder show button with text.

File: src/index.js (hunk 4 above)
Comment (hrach-o) [reply src/index.js:37]: If the initial data is not defined.

**Pull request merged**